source/controllers/index.md

```diff
@@ -1,14 +1,23 @@
 ## Controllers
```

Comment:
The changes to this file are great - if you break this into its own PR, I'd love to get it merged in straight away.